File: workflow-hello-java/src/main/java/com/acme/workflows/hello/java/Worker.java
package com.acme.workflows.hello.java;

import io.infinitic.workers.InfiniticWorker;

public class Worker {
    public static void main(String[] args) {
        try (InfiniticWorker worker = InfiniticWorker.fromConfigResource("/infinitic.yml", "/hello.yml")) {
            worker.start();
        }
    }
}

File: workflow-booking-kotlin/src/main/kotlin/com/acme/workflows/booking/kotlin/Worker.kt
package com.acme.workflows.booking.kotlin

import io.infinitic.workers.InfiniticWorker

fun main() {
    // infinitic.yml resource is from the dependant "contracts" module
    InfiniticWorker.fromConfigResource("/infinitic.yml", "/booking.yml").use { worker ->
        worker.start()
    }
}
